ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-console.j2

```diff
@@ -0,0 +1,17 @@
+{# Management Console #}
+{% if management_console is arista.avd.defined %}
+
+## Management Console
+{%      if management_console.idle_timeout is arista.avd.defined %}
```

Comment:
jinja-lint indentation: 5 spaces expected but found 6. We indent 4 spaces but you added an extra space for the indentation of ```if```.

Reply:
thanks Tony, happened by accident. Is there a way to test if basic codestyle is according to guidelines?